### ISSUE TITLE
HMIP-FSM16/HMIP-FSI16/HMIP-BSM/HMIP-FDT won't detected as light until init click

### ISSUE THREAD

**Issue by morpheus12** [SukramJ/hahomematic#1601]
### I agree to the following

- [X] I have read the [documentation](https://github.com/danielperna84/custom_homematic#custom_homematic) right now
- [X] I have read the [FAQs](https://github.com/danielperna84/custom_homematic#frequently-asked-questions) right now
- [X] I am aware of the latest [release notes](https://github.com/danielperna84/custom_homematic/releases)
- [X] I am aware that an integration that was installed via HACS also needs to be updated via HACS
- [X] The Backend (CCU/Homegear/...) is working as expected, devices are taught in and are controllable by its own UI.
- [X] I am running the latest version of the custom_component (and Home Assistant)

### The problem

Hi Team,
first great Plugin! I'm using it in 3 instances but here first time with higher than 1.61.XX
Its about a holiday home of my aunt. Here i have a ccu3 which is handling about 50 HMIP components and 3 HM-RF (the old RGBW Controller).
Today she told me the home assistant app is not working. After login i saw all HM-Components are not reacting.
In the logs of ha i saw an entry about HM-IP-RF ist not reacting pong is not answering as expected.
After a few minutes in the CCU3 webui i got the same error message that some components are not reacting.
So i rebooted the CCU3. Afterwords all is looking good. But in HA my dashboards with mushroom cards are not working.

mhh...strange

Therefore i have taken a look into the overview dashboard of HA and figured out some components are syncing data like hmip-wth, the old HM RGBW Controller, but hmip-fsm16/fsi16 and hmip-bsm/hmip-fdt are looking weird
![image](https://github.com/danielperna84/hahomematic/assets/23457727/33687c0b-bca9-4db4-8bac-3da7aeab9a70)

They looks like a power-plug/switch initally with the flash-symbol. For instance "Deckenlicht_Kueche" (FSM16)
After switching them in the overview dashboard to on the appearence change to a light switch like "Licht_Kueche_Arbeitsflaeche" (FSM16).

![image](https://github.com/danielperna84/hahomematic/assets/23457727/45a75955-a7f9-4a1d-88dc-3b72f7306f39)
Here i've done this with "Deckenlicht_Kueche". As you can see now the symbol switched to a "switch". This update take effekt within 3-4sec.

After they switched here in the overview-dashboard to a "light-switch" they're working in my own created mushroom dashboards (mushroom light card is used for it) as expected.
![image](https://github.com/danielperna84/hahomematic/assets/23457727/8b8cdbc2-33a8-4dfa-99ae-00937f36c801)

I assume the mushroom cards are not working because the entity is not "flagged correctly" as light until i click them initially in the overview dashboard to on.

I've restarted Home assistant to see the behavior (currently HA Core 2024.6.4, HA-OS 12.3).
![image](https://github.com/danielperna84/hahomematic/assets/23457727/8655bb54-9e81-4e41-8e51-60fe3d3ba3e8)
So the "already clicked" FSM/FSI actors are marked correctly. So i guess the problem is related to the ccu3 restart before.

Performing a manual relod/init of the plugin to update the ccu-data doesn't change anything.

Interesting topic: some of the actors are "enhanced" by an switch-adoption helper (to combine them as group with dimmers or the old hm rgbw controller).
This once are working fine from the beginning.

But i saw one hmip-fdt which is working fine from the beginning but others not.

Perhaps there is a kind of self-healing or correct initiation.
I will keep the last "incorrect" actors in this state and will have a look tomorrow on it. I will keep you up-to-date

thanks for reading until here.

### What version of HomematicIP (local) has the issue?

1.63.1

### What was the last working version of HomematicIP (local)?

assumption: 1.60

### What type of installation are you running?

Home Assistant OS

### What type of installation are you running for your homematic backend?

CCU3

### Which version of your homematic backend are you running?

3.61.7

### What hardware are you running for your system?

Intel Nuc

### Which config details do you use

- [ ] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [X] Homematic IP
- [X] Homematic (Bidcos-RF)
- [ ] Groups (Heating-Group)
- [ ] BidCos-Wired (HM-Wired)

**Comment by morpheus12** [comment]
Short Update: after about 1h all actors which wheren't clicked manually are now self-healed

**Comment by SukramJ** [comment]
TL;DR

> In the logs of ha i saw an entry about HM-IP-RF ist not reacting pong is not answering as expected.

Please add logs (Full logs) and Integration diagnostics.

**Comment by morpheus12** [comment]
[config_entry-homematicip_local-73cfb8cf40e7284e9fd8825ef3780362.json](https://github.com/user-attachments/files/15952290/config_entry-homematicip_local-73cfb8cf40e7284e9fd8825ef3780362.json)

Here i got the Integration Diagnostics.

The HA Log (i'm a idiot) is deleted. Since 2 days no real entries in the log file of HA.
`2024-06-22 22:41:14.154 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-06-22 22:41:14.160 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration homematicip_local which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2024-06-22 22:41:18.694 WARNING (MainThread) [homeassistant.helpers.frame] Detected that custom integration 'hacs' accesses hass.components.frontend. This is deprecated and will stop working in Home Assistant 2024.9, it should be updated to import functions used from frontend directly at custom_components/hacs/frontend.py, line 68: hass.components.frontend.async_register_built_in_panel(, please create a bug report at https://github.com/hacs/integration/issues
2024-06-22 22:41:18.843 WARNING (MainThread) [hass_nabucasa.cloud_api] Fetched https://alexa-api.nabucasa.com/access_token (400)
2024-06-22 22:41:27.492 WARNING (MainThread) [homeassistant.helpers.frame] Detected that custom integration 'hacs' accesses hass.helpers.event. This is deprecated and will stop working in Home Assistant 2024.11, it should be updated to import functions used from event directly at custom_components/hacs/base.py, line 642: self.hass.helpers.event.async_track_time_interval(, please create a bug report at https://github.com/hacs/integration/issues
`

Homematic is working fine since the last reboot.

**Comment by SukramJ** [comment]
> Homematic is working fine since the last reboot.

Sounds like a local communication problem and not like an issue with this integration.
So i'll close this, because only the log contains the relevant information.
